luna/featurevis/transformations.py

```diff
@@ -152,10 +293,98 @@ def color_augmentation(image, color_aug):
         color_aug (bool): True if image needs to augment.
 
     Returns:
-        img (list): Augmented image.
+        list: Augmented image.
     """
     if color_aug:
         image = tf.image.random_saturation(image, 0, 0.5)
         image = tf.image.random_contrast(image, 0, 0.8)
         image = tf.image.random_brightness(image, 0.01)
     return image
+
+
+def standard_transforms(img):
+    """Standard transformations if no transformations were chosen by user (Suggested by Lucid)
+
+    Args:
+        img (list): Image
+
+    Returns:
+        list: transformed imaged with standard modes
+    """
+    img = pad(img, 12, pad_mode="constant")
+    img = jitter(img, 8)
+    img = bilinear_rescale(img, [1 + (i - 5) / 50.0 for i in range(11)], seed=None)
+    img = rotation(img, list(range(-10, 11)) + 5 * [0])
+    img = jitter(img, 4)
+    return img
+
+
+def perform_trans(img, trans_param):
```

Comment:
This should have another name. Why does this preset exist?

Reply:
I changed the name of the variable to trans_func (after functionalizing the input). What exactly do you mean by "preset"?
Right now, this variable would be a function.